src/types/models/notifications.ts

```diff
@@ -0,0 +1,20 @@
+import { NotificationCategory } from '../../enum';
+import { SubCollectionOf, Timestamp } from '../utility';
+import { NotificationUserPreview } from './users';
+
+export type NotificationDocumentData = {};
+
+export type UserNotificationDocumentData = SubCollectionOf<'notifications', {
+   category: NotificationCategory;
+   message: string;
+   createdAt: Timestamp;
+   hasRead: boolean;
+   creator: NotificationUserPreview;
+}>;
+
+export type PushNotificationMessage = {
+   app_id: string;
+   contents: { en: string };
```

Comment:
what other contents can we send?
is this customizable?

we would want to also send the notificationId so that deep linking can refer to this notification. 

or possibly the messageId so deep linking can link to this message

Reply:
I have added more content. From the docs, its not customisable, I would look into how to pass custom data
https://documentation.onesignal.com/reference/create-notification